### DISCUSSION TITLE
Can not import name pymatgen.io.vasp.inputs

### DISCUSSION THREAD

**Discussion by bsantu** [materialsproject/pymatgen#3533] [Q&A]
Hi, everyone! While trying to import pymatgen, it doesn’t show any errors. However, when attempting to import pymatgen.io.vasp.inputs, I encounter an error titled ‘ValueError: dictionary update sequence element #0 has length 1; 2 is required.’ Could you please suggest some solutions for this error? Am I missing something? Below are some details about the versions I am using.

Pymatgen Version: 2023.12.18
Python Version: 3.11.5
Numpy Version: 1.26.2
Scipy Version: 1.11.4
conda 23.11.0

**Comment by janosh** [comment]
Please share the code you're running

**Reply by bsantu** [reply]
I used the following command to install Pymatgen with conda.
conda install --channel conda-forge pymatgen

**Reply by janosh** [reply]
I meant the python code you're trying to run.

**Reply by bsantu** [reply]
Hi, I am not running any particular Python code. Please have a look below.
(base) C:\Users\SANTU-BISWAS>python
Python 3.11.5 | packaged by Anaconda, Inc. | (main, Sep 11 2023, 13:26:23) [MSC v.1916 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import pymatgen
>>> import pymatgen.io.vasp.inputs
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\SANTU-BISWAS\anaconda3\Lib\site-packages\pymatgen\io\vasp\__init__.py", line 10, in <module>
    from .inputs import Incar, Kpoints, Poscar, Potcar, PotcarSingle, VaspInput
  File "C:\Users\SANTU-BISWAS\anaconda3\Lib\site-packages\pymatgen\io\vasp\inputs.py", line 34, in <module>
    from pymatgen.core import SETTINGS, Element, Lattice, Structure, get_el_sp
  File "C:\Users\SANTU-BISWAS\anaconda3\Lib\site-packages\pymatgen\core\__init__.py", line 79, in <module>
    locals().update(SETTINGS)
ValueError: dictionary update sequence element #0 has length 1; 2 is required
>>>

**Reply by janosh** [reply]
Your `.pmgrc.yaml` is compromised. Open it in your text editor and look for keys without values. If you can't find the issue, try starting from a fresh `.pmgrc.yaml`.

**Reply by bsantu** [reply]
Thank you so much! It is working now.
